front/app/containers/Admin/reporting/components/ReportBuilder/Widgets/ProjectsWidget/ProjectsCard/utils.ts

```diff
@@ -17,4 +21,46 @@ export const deriveProjectStatus = (period: Period, now: Moment) => {
   return now.isAfter(endAt) ? 'finished' : 'active';
 };
 
+export interface ISortOption<T> {
```

Comment:
Since we are defining the type only for this use case, no need to use a ts generic, you can directly define the value as `ProjectSortableParam`